java/simulator/src/main/java/com/hazelcast/simulator/worker/testcontainer/TestContainer.java

```diff
@@ -193,6 +193,20 @@ public void invoke(TestPhase testPhase) throws Exception {
                 throw e;
             }
         } finally {
+            // after the run phase we check if any negative latencies have
+            // been encountered.
+
+            if (testPhase == RUN) {
+                for (LatencyProbe probe : testContext.getLatencyProbes().values()) {
+                    if (probe.negativeCount() > 0) {
+                        String msg = "HdrLatencyProbe [" + probe.name() + "] has encounter "
+                                + probe.negativeCount() + " negative measurements! Maybe there"
+                                + " is a clock problem.";
```

Comment:
```suggestion
                        String msg = MessageFormat.format("HdrLatencyProbe [{0}] has encounter {1} negative measurements! Maybe there is a clock problem.", probe.name(), probe.negativeCount());
```
Nit - personally I find paramaterized strings easier to read. Plus the numbers get formatted in a localised way (e.g. `59494344` vs `59,494,344`)